core-plugins/src/main/java/co/cask/hydrator/plugin/batch/source/FileSourceConfig.java

```diff
@@ -50,6 +50,7 @@
     "which indicates that no files will be filtered.";
   protected static final String FILESYSTEM_PROPERTIES_DESCRIPTION = "A JSON string representing a map of properties " +
     "needed for the distributed file system.";
+  protected static final String FILE_SOURCE_FORMAT_DESCRIPTION = "Format of the file source. Defaults to text.";
```

Comment:
also include what the possible valid options are.

'Format of the file. Must be 'text', 'parquet', or 'avro'. Defaults to 'text'.